src/lib/haptics.ts

```diff
@@ -13,26 +13,30 @@ const hapticImpact: ImpactFeedbackStyle = isIOS
   : ImpactFeedbackStyle.Light // Users said the medium impact was too strong on Android; see APP-537s
 
 export class Haptics {
-  static default() {
-    if (isWeb) {
+  static default(disabled: boolean) {
+    if (disabled || isWeb) {
       return
     }
+
     impactAsync(hapticImpact)
   }
-  static impact(type: ImpactFeedbackStyle = hapticImpact) {
-    if (isWeb) {
+  static impact(type: ImpactFeedbackStyle = hapticImpact, disabled: boolean) {
+    if (disabled || isWeb) {
       return
     }
     impactAsync(type)
   }
-  static selection() {
-    if (isWeb) {
+  static selection(disabled: boolean) {
+    if (disabled || isWeb) {
       return
     }
     selectionAsync()
   }
-  static notification = (type: 'success' | 'warning' | 'error') => {
-    if (isWeb) {
+  static notification = (
+    type: 'success' | 'warning' | 'error',
+    enabled: boolean,
```

Comment:
Why do we have `enabled` here but `disabled` for others?

Reply:
forgot to invert this one, but removed anyway since we don't use it.